tsdoc-config/src/TSDocConfigFile.ts

```diff
@@ -132,6 +138,75 @@ export class TSDocConfigFile {
     return this._tagDefinitions;
   }
 
+  public get supportForTags(): ReadonlyMap<string, boolean> {
+    return this._supportForTags;
+  }
+
+  /**
+   * Removes all items from the `tagDefinitions` array.
+   */
+  public clearTagDefinitions(): void {
+    this._tagDefinitions.length = 0;
```

Comment:
Does this really clear the array? Why not just

```suggestion
    this._tagDefinitions = [];
```

Reply:
It's the only way to clear a `readonly` array.  I think this is a better practice than replacing the object with `[ ]`.